evalml/tests/automl_tests/test_automl.py

```diff
@@ -585,6 +585,31 @@ def test_large_dataset_regression(mock_score):
         assert automl.results['pipeline_results'][pipeline_id]['cv_data'][0]['score'] == 1.234
 
 
+@patch('evalml.pipelines.RegressionPipeline.score')
```

Comment:
@christopherbunn please also mock `fit`, that will save us the fit time in `automl.search`